app/src/main/res/values/strings.xml

```diff
@@ -555,6 +555,9 @@
     <string name="please_wait_updating_group_status">"Updating group status, please wait... "</string>
     <string name="error_while_updating_group_status">Error while updating group status</string>
     <string name="error_while_creating_group">Error while creating group</string>
+    <string name="please_wait_updating_teller_status" translatable="false">"Updating Teller status, please wait..."</string>
```

Comment:
Remove the quotation mark from the text. And also I have asked to add `translatable=false` to the strings that are not related to UI or if they are constants. But these strings will be displayed on UI and they need to be translated. So please remove this attribute or make it true.

Reply:
That's really a silly mistake from my side 😅. I'll do the changes

Reply:
I have pushed the changes, you can have a look now.